src/build.rs

```diff
@@ -357,6 +357,12 @@ impl BuildQueue {
                                 cmd_dep_info.arg(a);
                             }
                         }
+                        // Compilation may depend on env vars which Cargo sets during compile-time
+                        for (k, v) in cmd.get_envs() {
```

Comment:
Thanks for suggestion, didn't know how I could use filter_map with Option being one of tuple vars